mlflow/models/utils.py

```diff
@@ -27,6 +28,10 @@
 except ImportError:
     HAS_SCIPY = False
 
+INPUT_EXAMPLE_PATH = "artifact_path"
+PARAMS_EXAMPLE_PATH = "params_path"
+PARMAS_EXAMPLE_FILE = "params_example.json"
```

Comment:
I made a rethink on whether we should use the same file or not for storing the params example but didn't find any strong advantages of doing so.

```json
{
  "data": ...,
  "params": ...
}
```

This allows users to see both input data and params in one place, which sounds like an advantage.

Reply:
From mlflow artifact resource consumer perspective (e.g. mlflow, MLflow UI, model serving), separated files are much easier to handle.

Reply:
The only thing to validate is if the existing input_example.json is consumed by any other service / integration where a direct mutation of the structural contract might create a problem. 
If we're going with this approach, can we validate by logging a model to a '/Volumes/' path on UC, register the model, and submit it for serving to make sure that there are no integration issues with the change?

I also agree that this is more intuitive and useful for UX purposes :) Let's just be extra careful with validating any modifications to the structure of the file.

Reply:
Verified logging the model in UC with input_example and params_example, serving works fine.
<img width="619" alt="image" src="https://github.com/mlflow/mlflow/assets/82044803/982089d2-85da-4dca-a52a-f981455636b9">

<img width="861" alt="image" src="https://github.com/mlflow/mlflow/assets/82044803/4e6b6747-b494-436d-8d3d-e71e9f45b226">

But I didn't find a place that shows the files that saved for the model (input_example.json...) in UC. While looks like '/Volumes/...' is not a valid artifact_path for logging a model as it expects relative path.